src/util.rs

```diff
@@ -53,7 +52,7 @@ fn field_accessor(cx: &ExtCtxt, ident: &Spanned<ast::Ident>, mode: Accessor) ->
         Accessor::Set => "set_"
     };
 
-    respan(ident.span, cx.ident_of(&(prefix.to_string() + ident.node.as_str())))
+    respan(ident.span, cx.ident_of(&(prefix.to_string() + &ident.node.to_string())))
```

Comment:
This makes a useless copy. Use `ident.node.name.as_str()` instead.
Lots of `Ident` in the code could be replaced by `Name`. But that's fine for now.